_includes/trackers_get_response.json

```diff
@@ -0,0 +1,27 @@
+{
+  "id": "4a6922e2-09ad-4724-807c-7b4e572d3c6b",
```

Comment:
Should we sort this alphabetically as well?

And if `to` and `from` addresses are given, they will be returned as well.